crates/vm/levm/src/opcode_handlers/bitwise_comparison.rs

```diff
@@ -6,10 +6,33 @@ use crate::{
     vm::VM,
 };
 use ethrex_core::U256;
+use std::collections::HashMap;
+use std::sync::LazyLock;
 
 // Comparison and Bitwise Logic Operations (14)
 // Opcodes: LT, GT, SLT, SGT, EQ, ISZERO, AND, OR, XOR, NOT, BYTE, SHL, SHR, SAR
 
+static SHL_PRECALC: LazyLock<HashMap<u8, U256>> = LazyLock::new(|| {
```

Comment:
Why does this need to be static? Can't it be const?

Reply:
afaik for Hashmaps you need static (for instance https://users.rust-lang.org/t/is-there-a-way-to-create-a-constant-map-in-rust/8358) but if you have an idea I can try it

Reply:
I just checked with a match and whilst the code is a bit simpler and more direct; benchmarking I get that this current version is faster, so I dunno what we prefer in this situation.